monkestation/code/modules/projectiles/projectile/special/rocket.dm

```diff
@@ -0,0 +1,18 @@
+/obj/item/projectile/bullet/SRN_rocket
+	name = "SRN rocket"
+	icon = 'monkestation/icons/obj/guns/projectiles.dmi'
+	icon_state = "srn_rocket"
+	hitsound = "sound/effects/meteorimpact.ogg"
+	damage = 20
```

Comment:
```suggestion
	damage = 10
```
I know I suggested 20, but I didn't realize how fast the launcher fired.
Lets not turn this into a meta weapon.